test/domain/test_prepare_processing.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+from datetime import datetime
+from unittest.mock import MagicMock, patch
+
+from flexprep.domain.prepare_processing import PrepProcessing
+
+
+def test_aggregate_s3_objects():
+
+    objects = {
+        "Contents": [
+            {"Key": "P1S06180000061800011"},
+            {"Key": "P1S06180300061803001"},
+            {"Key": "P1D06180000061800001"},
+        ]
+    }
+
+    prep_processing = PrepProcessing()
```

Comment:
PrepProcessing should not be a class since it does not hold any state. The module is enough in terms of organization. Please create a ticket to address this in another PR.

Reply:
Here the PR where I refactored `prep_processing` into a module: https://github.com/MeteoSwiss-APN/flexprep/pull/3